.github/workflows/hardhat-core-ci.yml

```diff
@@ -118,6 +118,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
+        # TODO: Currently there is no @types/node for Node 22,
+        # include it when it is available.
         node: [18, 20]
```

Comment:
I believe the types for node v22 will only become available once it is in Active LTS